lib/target_mvd/targets.ex

```diff
@@ -47,9 +47,16 @@ defmodule TargetMvd.Targets do
   end
 
   def create_target(attrs \\ %{}) do
-    %Target{}
-    |> Target.changeset(attrs)
-    |> Repo.insert()
+    user = User |> Repo.get!(attrs["user_id"]) |> Repo.preload(:targets)
```

Comment:
Huh, never liked to restrict usage to either atoms or strings, @mcousillas6 @Alxzu any ideas? I've generally created a utility to extract options from maps